Makefile

```diff
@@ -19,17 +19,22 @@ CONFIG_LOCATION := $(SRC_DIR)
 $(@info $(shell mkdir -p $(OUT_DIR) $(BIN_DIR) $(PLUGIN_DIR))
 
 .PHONY: build
-build: proto_files cli server
+build: proto_files cli server executor
 
 .PHONY: proto_files
```

Comment:
When we are pushing .pb.go files, do we really need to have protogen in makefile?

Reply:
I think it's fine to have it in the makefile, so others who develop the proto could run Makefile to regenerate the .pb.go